x/dispensation/keeper/executors.go

```diff
@@ -31,8 +31,13 @@ func (k Keeper) CreateDrops(ctx sdk.Context, output []banktypes.Output, name str
 
 // DistributeDrops is called at the beginning of every block .
```

Comment:
Is it actually run at the beginning of every block? I couldn't find that logic.

Reply:
No its not , that was the older design , ill remove the comment